contracts/src/v0.8/test/HeartbeatRequester.t.sol

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+
+import "forge-std/Test.sol";
+import {HeartbeatRequester} from "../HeartbeatRequester.sol";
+import "../mocks/MockAggregator.sol";
+import "../mocks/MockAggregatorProxy.sol";
+import "../mocks/MockOffchainAggregator.sol";
+
+// from contracts directory,
+// forge test --match-path src/v0.8/test/HeartbeatRequester.t.sol
```

Comment:
Use Foundry profiles for this, for examples check how other teams have done this in foundry.toml. 

Then run `export FOUNDRY_PROFILE=automation` to activate that profile and you can run just automation tests with `forge test`.